src/include/OpenImageIO/imageio.h

```diff
@@ -2527,6 +2527,9 @@ class OIIO_API ImageOutput {
     ///        Does this format allow 0x0 sized images, i.e. an image file
     ///        with metadata only and no pixels?
     ///
+    ///  - `"cicp"` :
+    ///        Does this format support embedding CICP metadata?
+    ///
```

Comment:
Does a similar comment need to go in ImageInput::supports so that it's documented for both?

Reply:
It sure does! On it.

Reply:
need to make sure I've appropriately documented in oiiotool.rst and stdmetadata.rst as well.